packages/driver/src/cy/commands/xhr.js

```diff
@@ -471,6 +471,13 @@ module.exports = (Commands, Cypress, cy, state, config) => {
           })
         }
 
+        // look ahead to see if fixture exists
+        const fixturesRe = /^(fx:|fixture:)/
+
+        if (hasResponse && fixturesRe.test(options.response)) {
+          return cy.fixture(options.response.replace(fixturesRe, '')).then(() => route())
```

Comment:
Ok, I know this fragment makes an error in creating an alias for a route (for example in kitchen-sink example app) :( ... I need to fix it somehow.

@jennifer-shehane / @bahmutov 
Could you tell me if this kind of calling one command inside another is a good idea? Or maybe it would be better to directly call `backend` fixture?